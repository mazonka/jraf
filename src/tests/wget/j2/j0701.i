command=jw md 0 / : save 0 /c1 0 : put 0 /c2 0 0 : read 0 /c1 : read 0 /c2 : read 0 /