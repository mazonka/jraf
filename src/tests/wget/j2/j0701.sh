#!/bin/sh

cwd=`sh execdir.sh`
wr=$cwd/wroot
echo ""
echo "=="
ls -1 $wr
rm -rf $wr



