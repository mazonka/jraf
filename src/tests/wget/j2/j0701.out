OK  : OK 0 : OK 0 : OK 1 0  : OK 1 0  : OK 3 -1 2 1 0 c1 1 0 c2
==
c1
c2
